www/src/css/lib/vanilla-overrides.scss

```diff
@@ -5,4 +5,50 @@
 // If any styles from vanilla framework need to be overridden, corresponding
 // SCSS rules should be put in this file (not mixed with other styles).
 
+// scss/modules/_header.scss
+.banner .nav-primary ul li:hover {
+  background-color:#f7f7f7;
+}
 
+.banner .nav-primary ul li a,
+.banner .nav-primary ul li a:link,
+.banner .nav-primary ul li a:visited {
+  background-color:transparent;
+  font-size:initial;
+  border-left:0;
+  padding:14px 12px 8px 12px;
```

Comment:
Tdlr: without the overrides please. Always err on the side of Vanilla styles.

The only overrides we really want (not landing with this) is the addition of the icons next to the menu labels which 'I think' are still in the designs. To answer this question though, if it's this close to Vanilla it's almost 100% likely we were trying to use Vanilla styles but didn't get it right.

If we're not happy (in the design team) with margins / padding etc then we should change Vanilla and have the changes propagate to the different products.

@kwm14 I can catch you up on this when you're back… 